Tarballs are generated from the merged GIT repo now, so we need to
ensure we have a pure tag name as the version number, not some fancy
git commit description :-)
--- new-nav-debian.orig/version.sh
+++ new-nav-debian/version.sh
@@ -9,7 +9,7 @@ ingitrepo() {
 }
 
 do_describe() {
-  ${GIT} describe --tags
+  ${GIT} describe --abbrev=0
 }
 
 get_version() {
